python/meterpreter/ext_server_stdapi.py

```diff
@@ -1220,8 +1220,10 @@ def stdapi_fs_chdir(request, response):
 
 @register_function
 def stdapi_fs_delete(request, response):
-    file_path = packet_get_tlv(request, TLV_TYPE_FILE_NAME)['value']
-    os.unlink(unicode(file_path))
+    file_path = unicode(packet_get_tlv(request, TLV_TYPE_FILE_NAME)['value'])
+    if has_windll:
+        subprocess.call(unicode("attrib.exe -r ") + file_path)
+    os.unlink(file_path)
     return ERROR_SUCCESS, response
```

Comment:
This does not work in my testing; when I delete a ro file, it calls `stdapi_fs_delete_file`, not `stdapi_fs_delete` (this function).
If I move this syscall of attrib.exe to `stdapi_fs_delete_file` on line 1241, it works great.  Offhand, I'm unclear on the difference between the two, so I'm not sure if it should be moved or copied.

Reply:
FYI, @bcook-r7 I sent a PR to your branch with the fix, though it is only 2 lines.